MaterialDesignThemes.Wpf/DialogHost.cs

```diff
@@ -181,6 +181,22 @@ public static async Task<object> Show(object content, object dialogIdentifier, D
             return await targets[0].ShowInternal(content, openedEventHandler, closingEventHandler);
         }
 
+        /// <summary>
+        ///  Close a modal dialog.
+        /// </summary>
+        /// <param name="dialogIdentifier"> of the instance where the dialog should be shown. Typically this will match an identifer set in XAML. </param>
```

Comment:
```suggestion
        /// <param name="dialogIdentifier"> of the instance where the dialog should be closed. Typically this will match an identifer set in XAML. </param>
```